mods/default/functions.lua

```diff
@@ -293,13 +293,41 @@ minetest.register_abm({
 -- Dig upwards
 --
 
-function default.dig_up(pos, node, digger)
+local in_dig_up = false
+
+---Find all nodes above this one that is the same, then dig them all
+---@param pos vector The position of the base node
```

Comment:
(btw do we want to set a precedent of adopting LDoc?)

Reply:
I added LDoc because this function did not exist in `game_api.txt`. Should I move them there?

Reply:
Done in 45ef266667e41a6f3b2085c89e1f7126363be1ab